kotlinx.fuzz.jazzer/build.gradle.kts

```diff
@@ -8,15 +11,63 @@ repositories {
 
 dependencies {
     implementation(project(":kotlinx.fuzz.api"))
+    implementation("com.code-intelligence:jazzer:0.0.0-dev")
```

Comment:
Could you define jazzer dependency in `gradle/libs.versions.toml`?

Reply:
Fixed